intents/user/intentInitStudentInfoEN.json

```diff
@@ -0,0 +1,10 @@
+[
+    "Hi! I am [StudX]. I am a student in group [Gx], series [Sx], semigroup [SmX]. The current semester is [SemX] and the week is [WTx].",
+    "Hello! My name is [StudX]. I am a student in group [Gx], series [Sx], semigroup [SmX]. The semester is [SemX], and the week is [WTx].",
+    "Hi! My name is [StudX], student in group [Gx], series [Sx], semigroup [SmX]. The current semester is [SemX], and the study week is [WTx].",
+    "Hi! I am [StudX], I am a student in the group [Gx] [SmX], series [Sx]. The current semester is [SemX] and the parity is [WTx].",
+    "Hi! My name is [StudX]. I am a student in the group [Gx] [Sx] [SmX]. It is the semester [SemX], and the week is [WTx].",
+    "Hello! My name is [StudX]. I am a student in group [Gx], series [Sx], semigroup [SmX]. The semester is [SemX], and the week is [WTx].",
+    "Hello! My name is [StudX]. I am a student in the group [Gx] [Sx] [SmX]. The semester is [SemX], and the week is [WTx].",
+    "Hello! I am [StudX], a student in the group [Gx] [Sx] [SmX]. The semester is [SemX], and the week is [WTx]."
```

Comment:
You don't annontate the specified entities correctly.  You can look at the other intentions defined by me (for example getStartTimeEvent). 
For more details you can check the documentation: https://rasa.com/docs/rasa/nlu-training-data/